gym_lowcostrobot/envs/lift_cube_env.py

```diff
@@ -221,6 +246,8 @@ def reset(self, seed=None, options=None):
         cube_rot = np.array([1.0, 0.0, 0.0, 0.0])
         robot_qpos = np.array([0.0, 0.0, 0.0, 0.0, 0.0, 0.0])
         self.data.qpos[:] = np.concatenate([cube_pos, cube_rot, robot_qpos])
+        self.done = False
+        self.success = False
```

Comment:
Use `is_success` instead (standard).
btw I don't think it's necessary to make it an attribute. Just return it as part of the info dict.